integration/bundle/artifacts_test.go

```diff
@@ -1,223 +0,0 @@
-package bundle_test
```

Comment:
This is already covered by corresponding acceptance tests in `acceptance/bundle/artifacts`

Reply:
Brought back tests here https://github.com/databricks/cli/pull/3442/commits/74aa4345f14046205fdf7a1adb977388d56db272